clusters/prodpublick8s.yaml

```diff
@@ -146,23 +146,6 @@ releases:
       - "../config/incrementals-publisher.yaml"
     secrets:
       - "../secrets/config/incrementals-publisher/secrets.yaml"
-  - name: keycloak # Admin: https://admin.accounts.jenkins.io/auth/admin
-    namespace: keycloak
-    chart: codecentric/keycloak
-    version: 18.4.3
-    timeout: 600
-    values:
-      - "../config/keycloak.yaml"
-    secrets:
-      - "../secrets/config/keycloak/db.yaml"
-      - "../secrets/config/keycloak/http.yaml"
-  - name: keycloak-public # User: https://admin.accounts.jenkins.io/auth/realms/jenkins/account
-    namespace: keycloak
-    chart: jenkins-infra/keycloak
-    version: 0.1.1
-    timeout: 150
-    values:
-      - "../config/keycloak-public.yaml"
```

Comment:
Maybe remove keycloak-public in a separate PR?

Reply:
I opeted for the same pattern as in https://github.com/jenkins-infra/kubernetes-management/pull/3945. We can split in 2 not a problem

Reply:
The deletion of keycloak-public is distinct from the migration of keycloak (admin), not exactly the same purpose.

As you like 🙂

Reply:
I would keep the deletion of the keycloak (admin) release from clusters/prodpublick8s.yaml in this PR in any case.

Reply:
Oh I see, another incoming update.

A 2nd PR only for removing the "public" would make sense for auditability, while "single" PR make sense for "migration" so both are OK for me.

Reply:
https://github.com/jenkins-infra/kubernetes-management/pull/3984 here you are